webpack.config.js

```diff
@@ -34,7 +36,8 @@ module.exports = /** @type {import('webpack').Configuration} */ ({
     chunkFilename: `js/[name].chunk${hashSuffix}.js`,
     sourceMapFilename: `js/[name]${hashSuffix}.js.map`,
     path: resolve(__dirname, 'public/packs'),
-    publicPath: devServerPort ? `http://localhost:${devServerPort}/packs/` : '/packs/',
+    publicPath:
+      devServerPort && isLocalhost ? `http://localhost:${devServerPort}/packs/` : '/packs/',
```

Comment:
Would prettier take away parens if we added them here?

```suggestion
      (devServerPort && isLocalhost) ? `http://localhost:${devServerPort}/packs/` : '/packs/',
```

Reply:
[Yes, it will](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEAHArgIwBsBLMABQEMYALTAXkwAoATOANwGU4AnNnsiNywAyYZmJoAMhDAVC1CBgCUmAPyYABtRgxsSAPT7CMuQoxIAJMFacefbgKEBffdgpgA1mn0bMSTADkru5e+gEA3CAANCAQ2DDE6MigFNzcEADulNwIaMggchkUAJ55MfjcIXAwHBQAtnCSxFBwyABmcmhw5ZWe1RxuYM0A5sgw3LjdIHB1+HDMrMySFFDDuBTDcABignVUCav5FLgwENEg2nWEAOrUxPBog3BcUGj3xGz3xflgaGUgzS6QjIlWGe3anSmACs0AAPDgjQhwACKuAg8AhhC6MTc3CB+XwFDmhHO2G4zRg12IzBoyAAHAAGHHpLrXSrYfJkuBAvjnACOaPgILieRQFDQAFoWvN5uccgLiDkQRtwUgOlipl06sQxhNNYiUYLWmrITEYESqTTqMgAExmyrEEirADCEDqqumaAArOdcF0ACpE0Xq7EgNiTACSUFYsA4YHJ8QAgtGODBikjMV0nE4gA) 😬